src/main/java/frc/robot/RobotContainer.java

```diff
@@ -182,4 +182,10 @@ private void configureButtonBindings() {
   public Command getAutonomousCommand() {
     return autoChooser.get();
   }
+
+  public void log3DPoses{
+    Pose3d[] mechanismPoses = new Pose3d[];
+    mechanismPoses[0] = pivot.get3DPose();
+    Logger.recordOutput("3D Poses", mechanismPoses);
+  }
```

Comment:
This isn't ever getting called, so nothing gets logged. Have the function return mechanismPoses and then put an @AutoLogOutput annotation on the function